### DISCUSSION TITLE
Import vulns templates

### DISCUSSION THREAD

**Discussion by pwnhun73r** [Syslifters/sysreptor#212] [Q&A]
Hi! I started to use this tool and it's amazing I think that is the best reporting framework. I have a question about is there a resource like vulndb repo to upload vulnerabilities templates. I was looking for templates about mobile, API and web findings but I can't found. Thank you and you are doing a great job with this tool

**Comment by MWedl** [comment]
Hi, we currently only have about 10 demo templates (see https://github.com/Syslifters/sysreptor/tree/main/demo_data/demo-templates). We do not have another public template repository other than that.

If you want to contribute to the demo templates by adding new ones, feel free to open a pull request :)
